### ISSUE TITLE
Accessibility budget is not working with latest release 16.10.0 

### ISSUE THREAD

**Issue by TusAuto** [sitespeedio/sitespeed.io#3311]
1. I have upgraded sitespeed to latest v 16.10.0. Before this we were using 15.9.0 version.
2. In  v15.9.0 , we able make pass fail based on accessibility budget metrics. 
3. But in latest v 16.10.0 , we are getting error for   **accessibility**  metrics. Error screenshot is attached.
4. Because of this we are unable to move forword.
5. Please fix the same as soon as possible.
![Accessibilty_error](https://user-images.githubusercontent.com/47890381/111159688-61251500-85bf-11eb-8555-439fb7813fd3.PNG)
![BudgetFile](https://user-images.githubusercontent.com/47890381/111159696-62eed880-85bf-11eb-9d83-6f97b2342508.PNG)

**Comment by soulgalore** [comment]
Hi @TusAuto see the changed section in 16.0: https://github.com/sitespeedio/sitespeed.io/blob/main/CHANGELOG.md#1600---2020-12-21

We dropped the support for accessibility in the Coach. The reason is that the advice was very very limited and its much better to run AXE-core using `--axe.enable`.
